src/utils/storage/index.ts

```diff
@@ -83,13 +85,15 @@ const StorageUtils = {
   otherProfileData: storageBuilder(StorageKeysEnum.OtherProfileData),
   lastPromptNotification: storageBuilder(StorageKeysEnum.LastPromptNotification),
   lastSelectedMenu: storageBuilder(StorageKeysEnum.LastSelectedMenu),
+  blockingStatus: storageBuilder(StorageKeysEnum.BlockingStatus),
+  blockedStatus: storageBuilder(StorageKeysEnum.BlockedStatus),
   clearAll: () => MMKVStorage.clearAll()
 };
 
 const clearAll = () => {
   const onboardingPassword = StorageUtils.onboardingPassword.get();
   MMKVStorage.clearAll();
-  StorageUtils.onboardingPassword.set(onboardingPassword);
+  StorageUtils.onboardingPassword.set(onboardingPassword || '');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `clearAll` function to set `onboardingPassword` to an empty string if it's null or undefined could potentially introduce bugs. If there are checks elsewhere in the code that rely on `onboardingPassword` being `null` or `undefined` to indicate that it hasn't been set, this change could cause those checks to fail. Please ensure that this change doesn't break any existing logic that relies on `onboardingPassword` being `null` or `undefined`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->